.pylintrc

```diff
@@ -10,7 +10,7 @@ ignore-paths=src/s2python/generated/
 # avoid hangs.
 jobs=1
 
-disable=missing-class-docstring,missing-module-docstring,too-few-public-methods,missing-function-docstring
+disable=missing-class-docstring,missing-module-docstring,too-few-public-methods,missing-function-docstring,no-member
```

Comment:
pylint is not smart enough to follow protocols like mypy. Mypy should cover all no-member issues so disable in pylint.